### DISCUSSION TITLE
framework folder

### DISCUSSION THREAD

**Discussion by LLIKuIIeP** [iBotPeaches/Apktool#3272] [Q&A] [answered]
Please tell me, my applications are not rebuilt, apparently because of "Алексей" in the framework path: C:\Users\Алексей\AppData\Local\apktool\framework
How to change the framework folder? Just drag and drop didn't help, is there some command? Need to change to C:\apktool
Application build errors:

I: Using Apktool 2.8.1
I: Checking whether sources has changed...
I: Checking whether sources has changed...
I: Checking whether sources has changed...
I: Checking whether sources has changed...
I: Checking whether sources has changed...
I: Checking whether resources has changed...
I: Building resources...
W: 08-15 19:26:23.705 8032 4448 W ziparchive: Unable to open 'C:\Users\�������\AppData\Local\apktool\framework\1.apk': Illegal byte sequence
W: 08-15 19:26:23.705 8032 4448 W zipro : Error opening archive C:\Users\�������\AppData\Local\apktool\framework\1.apk: I/O error
W: C:\apktool\market1\res\values\styles.xml:6: error: Error: No resource found that matches the given name: attr 'android:colorAccent'.
W:
W: C:\apktool\market1\res\values\styles.xml:11: error: Error retrieving parent for item: No resource found that matches the given name '@android:style/Animation'.
W:
W: C:\apktool\market1\res\values\styles.xml:12: error: Error: No resource found that matches the given name: attr 'android:windowEnterAnimation'.
W:
W: C:\apktool\market1\res\values\styles.xml:13: error: Error: No resource found that matches the given name: attr 'android:windowExitAnimation'.
W:
W: C:\apktool\market1\res\values\styles.xml:15: error: Error retrieving parent for item: No resource found that matches the given name '@android:style/Animation'.
W:
W: C:\apktool\market1\res\values\styles.xml:16: error: Error: No resource found that matches the given name: attr 'android:windowEnterAnimation'.
W:
W: C:\apktool\market1\res\values\styles.xml:17: error: Error: No resource found that matches the given name: attr 'android:windowExitAnimation'.
W:
W: C:\apktool\market1\res\values\styles.xml:20: error: Error: No resource found that matches the given name: attr 'android:windowEnterAnimation'.
W:
W: C:\apktool\market1\res\values\styles.xml:21: error: Error: No resource found that matches the given name: attr 'android:windowExitAnimation'.
W:
W: C:\apktool\market1\res\values\styles.xml:24: error: Error: No resource found that matches the given name: attr 'android:windowEnterAnimation'.
W:
W: C:\apktool\market1\res\values\styles.xml:25: error: Error: No resource found that matches the given name: attr 'android:windowExitAnimation'.
W:
W: C:\apktool\market1\res\values\styles.xml:29: error: Error: No resource found that matches the given name: attr 'android:editTextBackground'.
W:
W: C:\apktool\market1\res\values\styles.xml:33: error: Error: No resource found that matches the given name: attr 'android:forceDarkAllowed'.
W:
W: C:\apktool\market1\res\values\styles.xml:31: error: Error: No resource found that matches the given name: attr 'android:statusBarColor'.
W:
W: C:\apktool\market1\res\values\styles.xml:28: error: Error: No resource found that matches the given name: attr 'android:windowBackground'.
W:
W: C:\apktool\market1\res\values\styles.xml:30: error: Error: No resource found that matches the given name: attr 'android:windowDrawsSystemBarBackgrounds'.
W:
W: C:\apktool\market1\res\values\styles.xml:32: error: Error: No resource found that matches the given name: attr 'android:windowLightStatusBar'.
W:
W: C:\apktool\market1\res\values\styles.xml:39: error: Error: No resource found that matches the given name: attr 'android:windowBackground'.
W:
W: C:\apktool\market1\res\values\styles.xml:41: error: Error: No resource found that matches the given name: attr 'android:windowDisablePreview'.
W:
W: C:\apktool\market1\res\values\styles.xml:40: error: Error: No resource found that matches the given name: attr 'android:windowIsTranslucent'.
W:
W: C:\apktool\market1\res\values\styles.xml:45: error: Error: No resource found that matches the given name: attr 'android:fontFamily'.
W:
W: C:\apktool\market1\res\values\styles.xml:153: error: Error retrieving parent for item: No resource found that matches the given name '@android:style/Widget'.
W:
W: C:\apktool\market1\res\values\styles.xml:154: error: Error: No resource found that matches the given name: attr 'android:layout'.
W:
W: C:\apktool\market1\res\values\styles.xml:162: error: Error retrieving parent for item: No resource found that matches the given name '@android:style/Animation'.
W:
W: C:\apktool\market1\res\values\styles.xml:163: error: Error: No resource found that matches the given name: attr 'android:windowEnterAnimation'.
W:
W: C:\apktool\market1\res\values\styles.xml:164: error: Error: No resource found that matches the given name: attr 'android:windowExitAnimation'.
W:
W: C:\apktool\market1\res\values\styles.xml:166: error: Error retrieving parent for item: No resource found that matches the given name '@android:style/Animation'.
W:
............

**Comment by iBotPeaches** [comment, marked as answer]
You can redirect the framework path from doc note here - https://apktool.org/docs/cli-parameters#decoding-options

> -p, --frame-path <DIR>
